lib/actions/ui.js

```diff
@@ -209,15 +210,29 @@ export function moveTo(i) {
   };
 }
 
+function getEditCommand(shell, isWin) {
+  if (isWin && shell.includes('bash')) {
+    return [
+      ' nano .hyper.js',
+      'echo Attempting to open .hyper.js with nano'
+    ];
+  } else if (isWin) {
+    return [
+      ' start notepad "%userprofile%\\.hyper.js"',
```

Comment:
@rauchg i pretty much just copy/pasted all the commands, and since the linter didn't cause any problems i just left it like that.